.github/workflows/main.yml

```diff
@@ -431,7 +431,9 @@ jobs:
     # Checks for no_std support, ensure that crates can build on a no_std
     # target
     - run: rustup target add x86_64-unknown-none
-    - run: cargo check -p wasmtime --no-default-features --features runtime,gc,component-model
+    - run: |
+        cargo check -p wasmtime --no-default-features --features runtime,gc,component-model
+        cargo check -p cranelift-control --no-default-features
       env:
         CARGO_BUILD_TARGET: x86_64-unknown-none
```

Comment:
Mind splitting these into separate `run` jobs so it's more obvious which one failed when it fails?

Reply:
Sure thing, done!